### PR TITLE
Improve readme organization

### PR DESCRIPTION
**Pull request by jcfr**
* Move compiling instructions to a dedicated BUILD document
* Update README improving intro, "Installation" and "History" sections

### CODE REVIEW

**Review by NicerNewerCar — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,60 +1,24 @@
...
+In November 2020, the development was transitioned from bitbucket to GitHub. See https://github.com/BrownBiomechanics/Autoscoper/
 
-1. Clone the [bitbucket repository](https://bitbucket.org/xromm/autoscoper-v2/src/BA_Playground/).
-2. Create a build folder in the autoscoper folder, open CMake and use XCode as compiler.
-3. When recieve an error, modify the fields:
-	1. CMAKE_OSX_ARCHITECTURES recommended to set to x86_64
-	2. CMAKE_OSX_DEPLOYMENT_TARGET to 10.15 (or your mac_os version)
-	3. If recieved an error for Qt5_DIR, search for (Qt5Config.cmake) on your hard drive and write its location in the field.
-4. After generating the configured file, open XCode and compile the application
+From 2018 to 2020, [Dr. Bardiya Akhbari][bardiya] from Brown University maintained the sofware on bitbucket, worked on addressing the reported issues, further optimized the CUDA kernels and implemented support for Particle Swarm Optimization (PSO) algorithm and active feedback on normalized cross-correlation to improve the accuracy and speed of registration.
```
Comment (BardiyaAk) [inline README.md:17]:
```suggestion
From 2018 to 2020, [Dr. Bardiya Akhbari][bardiya] from Brown University maintained the software on bitbucket, worked on addressing the reported issues, further optimized the CUDA kernels, and implemented support for the Particle Swarm Optimization (PSO) algorithm to improve the accuracy and speed of registration.
```

File: README.md [hunk 2]
```diff
@@ -1,60 +1,24 @@
...
 
-# History of Autoscoper #
-Autoscoper 1 was developed by Andy Loomis (original CUDA version) and [Mark Howison (OpenCL)](https://bitbucket.org/mhowison/xromm-autoscoper).
+From 2014 to 2018, Autoscope v2 was developed at Brown University by [Dr. Benjamin Knorlein][knorlein]. Multi-bone tracking and batch processing features were added, a socket-based protocol was designed for interaction with Matlab, the build system was transtioned to CMake and the UI was transitioned from GTK to Qt5. The version 2 combined the sources of both the CUDA and OpenCL versions and allows usage of either one. Source code was organized at `https://bitbucket.org/xromm/autoscoper-v2`.
```
Comment (BardiyaAk) [inline README.md:19]:
```suggestion
From 2014 to 2018, Autoscoper v2 was developed at Brown University by [Dr. Benjamin Knorlein][knorlein]. Multi-bone tracking and batch processing features were added, a socket-based protocol was designed for interaction with third-party software (mainly Matlab), the build system was transtioned to CMake and the UI was transitioned from Qt4 and GTK to Qt5. Version 2 combined the sources of both the CUDA and OpenCL versions and allows usage of either one. The source code was organized at `https://bitbucket.org/xromm/autoscoper-v2`. 
```

**Comment by BardiyaAk** [comment]
I edited two sections, wasn't sure if I need to commit them, or if they need to be checked before I commit them. Please let me know what's the best practice.

**Comment by jcfr** [comment]
> Please let me know what's the best practice.

Since I would like to keep the first two original commit separate, I will do the following:
* commit using the GitHub interface
* checkout the updated branch locally
* squash the last three commit together using `git rebase --interactive HEAD~3`
* add you as co-author using `Co-authored-by: Bardiya Ak <bardiya.akhbari@gmail.com>` to the commit titled `DOC: Update README improving intro, "Installation" and "History" sections`


For the rebase interactive part, here are the steps:

```
git checkout improve-readme-organization
git pull jcfr improve-readme-organization
git rebase --interactive HEAD~3
```

Then, update the rebase instructions like this:

image

Allowing me to then add you as a co-author ....

image

```
$ git rebase --interactive HEAD~3
[detached HEAD 63e6805] DOC: Update README improving intro, "Installation" and "History" sections
 Date: Mon Aug 1 18:58:02 2022 -0400
 1 file changed, 24 insertions(+), 14 deletions(-)
 rewrite README.md (89%)
Successfully rebased and updated refs/heads/improve-readme-organization.
```

Last, I pushed forced the topic doing:

```
$ git push jcfr improve-readme-organization --force
Enumerating objects: 5, done.
Counting objects: 100% (5/5), done.
Delta compression using up to 8 threads
Compressing objects: 100% (3/3), done.
Writing objects: 100% (3/3), 1.99 KiB | 1.99 MiB/s, done.
Total 3 (delta 1), reused 0 (delta 0)
remote: Resolving deltas: 100% (1/1), completed with 1 local object.
To github.com:jcfr/Autoscoper.git
 + 4aa8bb7...5125e3d improve-readme-organization -> improve-readme-organization (forced update)
```

And we can see the last commit now list you as a co-author:

image

To learn about the `Co-authored-by:` commit message trailer feature, see https://docs.github.com/en/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors

**Pull request merged**